src/pages/docs/presence-occupancy/presence.mdx

```diff
@@ -244,18 +233,20 @@ channel
+Each unique [`clientId`](/docs/api/realtime-sdk#client-id) may only be present once when entering on behalf of another client as the unique identifier for each member in a presence set is the combined [`clientId`](/docs/api/realtime-sdk#client-id) and shared [`connectionId`](/docs/api/realtime-sdk/connection#id).
 
-A single <span lang="default">"@clientId@":/docs/api/realtime-sdk#client-id</span><span lang="ruby">"@client_id@":/docs/api/realtime-sdk#client-id</span><span lang="csharp">"@ClientId@":/docs/api/realtime-sdk#client-id</span> may be present multiple times on the same channel via different client connections. Ably considers these as different members of the presence set for the channel, however they will be differentiated by their unique connectionId. For example, if a client with ID “Sarah” is connected to a chat channel on both a desktop and a mobile device simultaneously, “Sarah” will be present twice in the presence member set with the same client ID, yet will have two unique connection IDs. A member of the presence set is therefore unique by the combination of the clientId and connectionId strings.
+A single [`clientId`](/docs/api/realtime-sdk#client-id) may be present multiple times on the same channel via different client connections. Ably considers these as different members of the presence set for the channel, however they will be differentiated by their unique connectionId. For example, if a client with ID “Sarah” is connected to a chat channel on both a desktop and a mobile device simultaneously, “Sarah” will be present twice in the presence member set with the same client ID, yet will have two unique connection IDs. A member of the presence set is therefore unique by the combination of the clientId and connectionId strings.
 
-In order to be able to publish presence changes for arbitrary client IDs, the SDK must have been instantiated either with an "API key":https://faqs.ably.com/what-is-an-app-api-key, or with a "token bound to a wildcard client ID":https://faqs.ably.com/can-a-client-emulate-any-client-id-i.e.-authenticate-using-a-wildcard-client-id.
+In order to be able to publish presence changes for arbitrary client IDs, the SDK must have been instantiated either with an [API key](https://faqs.ably.com/what-is-an-app-api-key), or with a [token bound to a wildcard client ID](https://faqs.ably.com/can-a-client-emulate-any-client-id-i.e.-authenticate-using-a-wildcard-client-id).
 
-```[rest_javascript]
+<Code>
```

Comment:
This isn't displaying correctly, but maybe doesn't have the rebased fix from before?

Reply:
[This](https://github.com/ably/ably-ui/pull/888) should sort it, have ran a review case past Greg already. Hang tight on the ably ui bump